### PR TITLE
MET-3240 Fix transformation xslt parameters

### PR DESCRIPTION
**Pull request by stzanakis**

### CODE REVIEW

File: metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/WorkflowExecutionFactory.java [hunk 1]
```diff
@@ -254,8 +254,7 @@ private void setupXsltIdForPluginMetadata(Dataset dataset,
     if (xsltObject != null && StringUtils.isNotEmpty(xsltObject.getXslt())) {
       pluginMetadata.setXsltId(xsltObject.getId().toString());
     }
-    //DatasetName in Transformation should be a concatenation datasetId_datasetName
-    pluginMetadata.setDatasetName(dataset.getDatasetId() + "_" + dataset.getDatasetName());
+    pluginMetadata.setDatasetName(dataset.getDatasetName());
```
Comment (jochen-vermeulen) [inline metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/WorkflowExecutionFactory.java:257]: We are sure that in eCloud this parameter is ONLY used as input for our validation service, right?

File: metis-data-checker/metis-data-checker-service/src/main/java/eu/europeana/metis/data/checker/service/executor/ValidationTask.java [hunk 2]
```diff
@@ -122,10 +122,10 @@ private String transformRecord() throws EuropeanaIdException, TransformationExce
     final EuropeanaGeneratedIdsMap europeanaGeneratedIdsMap = new EuropeanaIdCreator()
```
Comment (jochen-vermeulen) [inline metis-data-checker/metis-data-checker-service/src/main/java/eu/europeana/metis/data/checker/service/executor/ValidationTask.java:122]: This variable is no longer used.

**Review by jochen-vermeulen — CHANGES_REQUESTED** [review]
I have not completed my review. If this PR is to be merged, we'll have to look at it carefully again.

**Comment by stzanakis** [comment]
Not to be merged, xslt issues resolved on the xslt itself.
In the future changes on the xslt that require code changes should be reviewed beforehand.

**Pull request closed without merging**